Android/app/src/main/java/com/example/boardgamesocial/API/API.java

```diff
@@ -45,6 +46,7 @@
 
     HashMap<Class<?>, String> URL_MAP = new HashMap<Class<?>, String>() {{
         put(User.class, "api/user/");
+        put(Profile.class, "api/profile");
```

Comment:
I just came across a few things, this one is missing the trailing "/"
```suggestion
        put(Profile.class, "api/profile/");
```

Reply:
Ok, will add it 👍